code/datums/supply_packs/operations.dm

```diff
@@ -1,6 +1,99 @@
 //the idea is to put all the bulk items scanner secure crate with lot's of flares MRE in it and at the end OB and non buyable.
 
-//non buyable
+//large quantity of metals/sandbags/plasteel.x5 the normal crate and with a 10% discount
+
+/datum/supply_packs/metalsheetbulk
+	name = "metal sheets(x250)"
+	contains = list(
+					/obj/item/stack/sheet/metal/large_stack,
+					/obj/item/stack/sheet/metal/large_stack,
+					/obj/item/stack/sheet/metal/large_stack,
+					/obj/item/stack/sheet/metal/large_stack,
+					/obj/item/stack/sheet/metal/large_stack,
+					)
+	cost = 90
+	containertype = /obj/structure/largecrate
+	containername = "metal sheets crate (x250)"
+	group = "Operations"
```

Comment:
remove, I do not wish to make it easier to buy materials

Reply:
removed